src/axom/quest/Delaunay.hpp

```diff
@@ -74,7 +74,7 @@ class Delaunay
 public:
   /**
    * \brief Default constructor
-   * \note User need to call initializeBoundary(BoundingBox) before adding points.
+   * \note User must to call initializeBoundary(BoundingBox) before adding points.
```

Comment:
I know that I'm not on the axom team and this is a small thing, but you have a typo here "User must to call". Probably you just want to say "User must call", no 'to'.

Reply:
Thanks @dtaller !